### PR TITLE
Update to Tensorflow 1.13.1

### LINKED ISSUE #95: Update to TensorFlow 1.13
### Motivation

There is a [bug](https://github.com/tensorflow/tensorflow/issues/25175) in TensorFlow 1.12 which breaks dropout layers in Keras.

### PR DESCRIPTION
**Pull request by 1vn**
fixes #95
---
Checklist:
- [ ] Run `make test` and `make lint`
- [ ] Test end to end manually (e.g. build/push all images, restart local operator, and run `cx refresh` in an example folder)
- [ ] Update documentation
- [ ] Update examples and `cx init`
- [ ] Alert team if dev environment changed
- [ ] Cherry-pick into release branches if it's a bugfix
- [ ] Delete the branch once it's merged

### CODE REVIEW

File: images/tf-serve/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 FROM cortexlabs/tf-base
 
-ARG TF_VERSION="1.12.0"
+ARG TF_SERV_VERSION="1.12.0"
```
Comment (deliahu) [inline images/tf-serve/Dockerfile]: I went to their [releases](https://github.com/tensorflow/serving/releases) and noticed that they have `1.13.0`, so I tried that and it worked. Can you update to `1.13.0`?

File: images/tf-serve-gpu/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 FROM cortexlabs/tf-base-gpu
 
-ARG TF_VERSION="1.12.0"
+ARG TF_SERV_VERSION="1.12.0"
```
Comment (deliahu) [inline images/tf-serve-gpu/Dockerfile]: I went to their [releases](https://github.com/tensorflow/serving/releases) and noticed that they have `1.13.0`, so I tried that and it worked. Can you update to `1.13.0`?

File: examples/mnist/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
 pillow==5.4.1
-tensor2tensor==1.10.0
+tensor2tensor==1.12.0
```
Comment (deliahu) [inline examples/mnist/requirements.txt]: Their latest [release](https://github.com/tensorflow/tensor2tensor/releases) seems to be `1.13.4`, can we use that?

File: examples/reviews/requirements.txt [hunk 4]
```diff
@@ -1 +1 @@
-tensor2tensor==1.10.0
+tensor2tensor==1.12.0
```
Comment (deliahu) [inline examples/reviews/requirements.txt]: Their latest [release](https://github.com/tensorflow/tensor2tensor/releases) seems to be `1.13.4`, can we use that?
